.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: True
+contact_links:
+  - name: Documentation
+    url: https://github.com/Unity-Technologies/com.unity.multiplayer.docs/issues
```

Comment:
This will be 404 for external people for now but not for long so it's fine.

Reply:
Doc team will have news on that hopefully within a week or so.